src/components/application_manager/src/message_helper.cc

```diff
@@ -1102,11 +1102,11 @@ smart_objects::SmartObjectList MessageHelper::CreateAddCommandRequestToHMI(
       msg_params[strings::menu_params] = (*i->second)[strings::menu_params];
       msg_params[strings::app_id] = app->app_id();
 
-      if (((*i->second)[strings::cmd_icon].keyExists(strings::value))
+      if (((*i->second).keyExists(strings::cmd_icon))
```

Comment:
Could we move reference to the first and second value of pair at the beginning of the loop?
It will make code more readable.